app/components/MazemapEmbed/index.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import { useEffect } from 'react';
+import "node_modules/mazemap/mazemap.min.css"
+import * as Mazemap from 'mazemap';
+
+type Props = {
+  sharepoi: number,
+  height?: number,
+};
+
+/** A component that shows a mazemap map of a given poi (e.g. room),
+ * largely based on https://api.mazemap.com/js/v2.0.12/docs/#ex-data-poi
+ */
+
+export const MazemapEmbed = ({ sharepoi, ...props }: Props) => {
+  //useEffect to initialize only once, sharepoi will probably not change
+  useEffect(() => {
+    // $FlowFixMe Mazemap is defined in script tag, has no types
+    const myMap = new Mazemap.Map({
```

Comment:
Very minor, but I don't like variable names like `myMap` and `mySearch` 🙃 Maybe something that describes where this map is centered at, or what it aims to show?